test/e2e/packetcapture_test.go

```diff
@@ -586,6 +606,22 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 	if !packetCaptureStatusEqual(pc.Status, tc.expectedStatus) {
 		t.Errorf("CR status not match, actual: %+v, expected: %+v", pc.Status, tc.expectedStatus)
 	}
+
+	if tc.expectedStatus.NumberCaptured == 0 {
+		return
+	}
+	// verify packets.
+	antreaPodName, err := data.getAntreaPodOnNode(nodeName(0))
+	require.NoError(t, err)
+	fileName := fmt.Sprintf("%s.pcapng", tc.pc.Name)
+	dstFileName := filepath.Join(os.TempDir(), fileName)
+	packetFile := filepath.Join("/tmp", "antrea", "packetcapture", "packets", fileName)
+	require.NoError(t, data.copyPodFiles(antreaPodName, "antrea-agent", "kube-system", packetFile, os.TempDir()))
+	defer os.Remove(dstFileName)
```

Comment:
It might be better to use `t.TempDir()` which will be automatically deleted when the test completes? Otherwise `copyPodFiles` could fail and leave a file not cleaned.

Reply:
sure.